<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>whotel</class>
 <widget class="QWidget" name="whotel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>房间列表</string>
    </attribute>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>1051</width>
       <height>421</height>
      </rect>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btnChange">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>更改信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>40</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>保存退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefresh">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNew">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>新建房间</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>订单列表</string>
    </attribute>
    <widget class="QPushButton" name="btnCheckIn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认入住</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCheckOut">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认退房</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>1051</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefresh_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
